src/api/Shrooms.Domain/Extensions/DbSetExtensions.cs

```diff
@@ -11,8 +10,74 @@ public static class DbSetExtensions
     {
         private const string DefaultSortDirection = "asc";
```

Comment:
I'm starting to think that these extensions more likely belong to data access layer.
Another thing, that it looks like it extends `IQueryable` rather than `DbSet` (because of `this IQueryable<>` parameters everywhere).

Reply:
Shrooms.Domain.Services project does not have a reference to Shrooms.DataLayer.DAL (only Shrooms.DataLayer.EntityModels which is another project). Would you like me to add reference to DAL and move the extensions? (Also I renamed this from DbSetExtensions to QueryableExtensions)

Reply:
Ok, let's leave it in Service layer then.